src/Relation/Nullary/Decidable/Core.agda

```diff
@@ -104,8 +106,8 @@ proof (a? →-dec b?) = proof a? →-reflects proof b?
 -- for proof automation.
 
 isYes : Dec A → Bool
-isYes (true  because _) = true
-isYes (false because _) = false
+isYes (yes′ _) = true
+isYes (no′  _) = false
```

Comment:
So... this is claimed as a more strict version of `does`... even in the original version of this module. That suggest the following possible refactoring:
* in `Data.Bool.Base` add
```agda
strictify : Bool → Bool
strictify b = if b then true else false
```
* and then `isYes = strictify ∘ does`

Would that be preferable?

Reply:
I certainly find that idiom appealing.

Reply:
Perhaps that's worth considering for `data` generally, not just `Bool`? 
(I'm mostly think of `Nat`, but others perhaps also...)

Reply:
Yes, ought to consider strictification of various `data`. Too bad it doesn't seem like that idiom can be made generic (or, at least, my brain can't see how to do it, without meta-programming.)